modules/Core/pages/profile.php

```diff
@@ -108,6 +108,18 @@
                                 ]
                             );
 
+                            EventHandler::executeEvent('userNewProfilePost', [
+                                'username' => $user->getDisplayname(true),
+                                'content' => $language->get("user", "x_posted_on_y_profile", [
```

Comment:
This essentially doesn't contain the content though? Just more or less an alert text. I'd say the name of it doesn't fit in that case.

Reply:
Sorry, I don't quite understand what you mean. The 'content' field is just so that it ends up in the embed footer: https://github.com/NamelessMC/Nameless/blob/develop/modules/Discord%20Integration/hooks/DiscordHook.php#L30 
Unless you meant something else? Thanks.

Reply:
Ah right. Forgot it was used for that. Nvm my comment then